### PR TITLE
#181339490 profile page

### PR DESCRIPTION
**Pull request by Gabin-ishimwe**
## What does this PR do?
user can update their profile on barefoot nomad application
user can logout from our application
## Description of Task to be completed?
update profile on barefoot nomad application
logout of barefoot nomad
## How should this be manually tested?
[https://deploy-preview-18--barefoot-nomad-fe.netlify.app/login](https://deploy-preview-18--barefoot-nomad-fe.netlify.app/login)
## Any background context you want to provide?

- first login or register your account
- go on your profile update it

## What are the relevant pivotal tracker stories?
[https://www.pivotaltracker.com/n/projects/2555280/stories/181339490](181339490)
## Questions:

### CODE REVIEW

File: tests/profile.test.js [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+    fireEvent.change(lastName, { target: { value: 'ishimwe' } });
+    fireEvent.change(age, { target: { value: 19 } });
+    fireEvent.change(dateOfBirth, { target: { value: '02/02/2002' } });
+    fireEvent.change(occupation, { target: { value: 'software engineer' } });
+
+    fireEvent.click(submit);
+    await waitFor(() => {
+      expect(screen.getByTestId('spinner')).toBeInTheDocument();
+    });
+    // moxios.stubRequest("POST", "/users/", {
```
Comment (Tsa2341) [inline tests/profile.test.js:89]: remove commented code, do it in all the other files  also.

File: src/components/input.js [hunk 2]
```diff
@@ -1,42 +1,55 @@
...
+    width: 280,
+    height: 20,
+    margin: '30px 0px',
+  },
+  width: 350,
+  height: 50,
+  margin: '20px 0px',
+}));
+const InputField = ({ otherStyles, ...props }) => {
+  // const [value, setValue] = useState('');
```
Comment (Tsa2341) [inline src/components/input.js:17]: remove commented code, do it in all the others files also

File: src/redux/actions/location.action.js [hunk 3]
```diff
@@ -45,4 +46,26 @@ export const getLoc = () => async (dispatch) => {
...
+    payload: data,
+    loading: false
+  }
+}
+
+export const getOneLocation = (id) => {
+  return async (dispatch) => {
+    try {
+      const res = await axios.get(`/locations/${id}`)
+      console.log(res)
```
Comment (Tsa2341) [inline src/redux/actions/location.action.js:64]: try to remove all un needed console.logs

Comment (Tsa2341) [reply src/redux/actions/location.action.js:64]: do it also in all the other files also

**Review by Tsa2341 — CHANGES_REQUESTED** [review]
also check the date of birth input validattion.

and the email and app notification buttons functionality, when you unselect one all becomes unselected.

**Review by blandri — APPROVED** [review]

**Review by Code-250 — APPROVED** [review]

**Review by Code-250 — APPROVED** [review]

**Pull request merged**